### PR TITLE
es visitor: de should be range query

### PR DESCRIPTION
**Pull request by MJedr**
ref: inspirehep/inspirehep#2345

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,8 @@
     'pypeg2~=2.0,>=2.15.2',
     'python-dateutil~=2.0,>=2.6.1',
     'six~=1.0,>=1.11.0',
-    'datefinder~=0.7.1'
+    'datefinder~=0.7.1',
+    'regex~=2021.3.17'
```
Comment (michamos) [inline setup.py:45]: are you using this anywhere? or is it already used and you're pinning it for some reason?

Comment (MJedr) [reply setup.py:45]: yes, python2 is no longer supported, so build for python2 was failing

Comment (michamos) [reply setup.py:45]: OTOH, do we really need to support python2 here? the query parser is not actively used on `inspire-next` (it depends on it due to dead code, but that can probably stay pinned to an older version). @drjova?

Comment (drjova) [reply setup.py:45]: I agree, we can remove support of python 2 if it's correctly pinned in inspire-next

**Review by drjova — APPROVED** [review]

**Pull request merged**